autoendpoint/src/error.rs

```diff
@@ -1,6 +1,7 @@
 //! Error types and transformations
 
-use crate::server::{RouterError, VapidError};
+use crate::headers::vapid::VapidError;
+use crate::routers::RouterError;
```

Comment:
Probably also a nit, but we've previous used a python approach to library sections:

so includes are broken into groups of

* System/internal (e.g. `std`)
* External libraries (e.g. `actix::*`, usually in alphabetical order)
* Package internal (e.g. `crate::`) 

It's optional, and makes zero difference to the compiler or linker.

Reply:
I like the idea, but I've found that it doesn't work well with auto-importing. At least for IntelliJ Rust, the import will often be placed in the wrong block which messes up the groups and requires manual intervention.

Additionally, rustfmt will already order the imports nicely (crate first, then alphabetically). Leaving it up to rustfmt seems more productive to me, and the CI can check it.

Reply:
Ah, it makes sense if you're using the auto-import feature. That's why these are nits more than anything.